src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -1842,6 +1842,17 @@ public static class ProfileOptimization
         public static void StartProfile(string profile) { throw null; }
     }
 }
+namespace System.Runtime.CompilerServices
+{
+    [Serializable]
```

Comment:
We don't need `[Serializable]` in the ref. It's only used by the runtime, not the compiler.